Applications/TransferLearning/Draw_Classification/README.md

```diff
@@ -1,51 +1,65 @@
 # Classification of hand-drawn images
 
-Unlike using kNN for the classifier at #55 , we could add new layer at the end of feature extractor.
-In this toy example, fully connected layer is added to classify as below. The size of the feature is also same with previous example and the training set and test set are also same. ( 3 classes, 5 training set and 2 test set of each )
-Only fully connected layers are updatable and Mobilenet ssd v2 is used for feature extractor like previous example and all the testing and training is done on the Galaxy S8.
+Draw Classification example performs classification on the hand-drawn images and maps them to different emotions.
+The application uses a Mobilenet-V2 model pre-trained on ms-coco dataset.
+Transfer learning to used to take advantage of the pre-training model to learn the new information of the
+hand-drawn image dataset efficiently.
 
-The activation is softmax. Mean Squared Error loss function and Stochastic Gradient Decent is used for the loss function and optimizer. Two fully connected layers are added for the hidden layer and iterate 300 times to make is more simple.
+## Application Details
+
+Below are the steps to perform transfer learning in this application:
+
+1. The pre-trained model is converted from pb format (Tensorflow trained model file) to TFLite format (Tensorflow-Lite frozen model). The last layer of the model is removed in this conversion to allow transfer learning to be done.
+2. The training data is passed through the tflite model and the outputs cached. As the transfer learning only trains the last few layers, the outputs from the non-trainable layers are cached to save computation. This is achieved using the single-shot C-API of NNStreamer for TIZEN. In case of non-tizen OS, Tensorflow-Lite framework c++ API are directly used.
```

Comment:
the outputs cached -> the outputs are cached?

Reply:
done.